typescript/forms.d.ts

```diff
@@ -840,6 +840,26 @@ export type Form = {
      * when denying an approval
      */
     denyCannedResponses?: FormApprovalCannedResponse[]
+    /**
+     * Configuration for automatically denying an approval after a number of
+     * days when a clarification request has been sent with no response.
+     * Set `undefined` or unset for no Auto Deny.
+     */
+    autoDenyAfterClarificationRequest?: {
+      /**
+       * The amount of days after a clarification request has been sent with no
+       * response until the approval is automatically denied.
+       */
+      days: number
+      /** The email addresses of the users to be notified of the result */
+      notificationEmailAddress?: string[]
```

Comment:
This should a boolean to indicate if they want to send a notification (the address will be the default address if configured/available)

Reply:
Perhaps we shoud keep this field as well and add the default address to this later when sending?

Reply:
Hmm, feels like it should have the same options as the approvals front end

Reply:
Cool, we'll keep it then and add another field for choosing to notify